terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,350 @@
+   policy_arn = "${aws_iam_policy.backup_lambda_permissions.arn}"
+}
+
+
+resource "aws_iam_policy" "allow_backup_lambda_logging" {
+  count       = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+  name        = "BackupLambda-${var.network_id}-${var.aws_region}"
+  path        = "/"
+  description = "IAM policy for logging from a lambda"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "logs:*"
+      ],
+      "Resource": "arn:aws:logs:*:*:*",
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "null_resource" "mkdir_temp" {
+  triggers { always="${uuid()}" }
+  provisioner "local-exec" {
+    command = <<EOT
+  mkdir -p ${path.module}/tmp/
+EOT
+  }
+  provisioner "local-exec" {
+    when = "destroy"
+    command = "rm -rf ${path.module}/tmp"
+    on_failure = "continue"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "allow_backup_lambda_logging" {
+   count      = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+   role       = "${aws_iam_role.backup_lambda.name}"
+   policy_arn = "${aws_iam_policy.allow_backup_lambda_logging.arn}"
+}
+
+resource "null_resource" "fetch_backup_lambda_zip" {
+  count = "${var.backup_enabled ? 1 : 0}"
+
+  triggers { always="${uuid()}" }
+  provisioner "local-exec" {
+     command = <<EOT
+if [ ! -e ${local.temp_lambda_zip_path} ]; then 
+   wget -O ${local.temp_lambda_zip_path} ${var.backup_lambda_binary_url}
+fi
+EOT
+  }
+  provisioner "local-exec" {
+    when = "destroy"
+    command = "rm ${local.temp_lambda_zip_path}"
+    on_failure = "continue"
+  }
+
+  depends_on = ["null_resource.mkdir_temp"]
+}
+
+resource "aws_kms_key" "ssh_encryption_key" {
+  count       = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+  description = "Used for encrypting SSH keys on S3"
+
+  tags {
+    Name = "BackupLambda-${var.network_id}-${var.aws_region}-KMS"
+  }
+}
+
+# Encrypt the contents of the SSH key
+data "aws_kms_ciphertext" "encrypt_ssh_operation" {
+  count     = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+  key_id    = "${aws_kms_key.ssh_encryption_key.id}"  
+  plaintext = "${var.backup_lambda_ssh_private_key}"
+}
+
+resource "aws_kms_grant" "backup_lambda" {
+  count             = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+  name              = "kms-grant-${var.network_id}-${var.aws_region}"
+  key_id            = "${aws_kms_key.ssh_encryption_key.key_id}"
+  grantee_principal = "${aws_iam_role.backup_lambda.arn}"
+  operations        = ["Encrypt", "Decrypt"]
+}
+
+resource "aws_security_group" "allow_all_for_backup_lambda" {
+  count       = "${var.backup_enabled ? local.signum_lookup : 0}"
+
+  name        = "BackupLambdaSSH-${var.network_id}-${var.aws_region}-allow-all-outgoing"
+  description = "Allow all outgoing traffic for BackupLambda"
+  vpc_id      = "${aws_vpc.quorum_cluster.id}"
+
+  tags {
+     name = "BackupLambda-${var.network_id}-${var.aws_region}-SG"
```

Comment:
nit: I believe the `name` tag should be capitalized as `Name` to work with the AWS console

Reply:
Fixed.